theseus/extlib/test_mat_mult.py

```diff
@@ -0,0 +1,119 @@
+import pytest  # noqa: F401
+import torch  # needed for import of Torch C++ extensions to work
+from scipy.sparse import csr_matrix
+
+from theseus.extlib.sparse_matrix_utils import generate_sparse_matrix
+
+
+def check_mat_mult(batch_size, rows, cols, fill, verbose=False):
```

Comment:
Same as pervious PR about moving tests to a separate tests dir.

Reply:
done!